raiden/routing.py

```diff
@@ -338,7 +329,6 @@ def resolve_routes(
             resolvable.append(
                 RouteState(
                     route=route_metadata.route,
-                    forward_channel_id=channel_state.canonical_identifier.channel_identifier,
                     # This is only used in the mediator, so fees are set to 0
                     estimated_fee=FeeAmount(0),
                 )
```

Comment:
looks like we dont need this function anymore, but might be reintroduced if we use unique channel ids?